### PR TITLE
Generic operation handler update

### LINKED ISSUE #2487: Generic Operation

### PR DESCRIPTION
**Pull request by r32575**
Convert handlers to the new [Maestro](https://github.com/gchq/Maestro) format

# Related Issue

- Resolve #2487

### CODE REVIEW

File: store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java [hunk 1]
```diff
@@ -18,34 +18,49 @@
...
-    }
+public class AddElementsHandler implements OperationHandler<Void> {
+
+    private static final String SKIP_INVALID_ELEMENTS = "skipInvalidElements";
 
-    private void addElements(final AddElements operation, final AccumuloStore store)
-            throws OperationException {
+    private static final String VALIDATE = "validate";
+
+    private static final String INPUT = "input";
```
Comment (GCHQDev404) [inline store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java:36]: input isn't required.

Comment (GCHQDev404) [inline store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java:36]: No longer required.
If you Pull my branch update into yours.

File: store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java [hunk 2]
```diff
@@ -18,34 +18,49 @@
...
+            return null;
         } catch (final StoreException e) {
             throw new OperationException("Failed to add elements", e);
         }
     }
+
+    @Override
+    public FieldDeclaration getFieldDeclaration() {
+        return new FieldDeclaration()
+                .fieldRequired(INPUT, Iterable.class)
```
Comment (GCHQDev404) [inline store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java:62]: If you Pull my branch update into yours.
.inputRequired(Iterable.class)

**Review by GCHQDev404 — CHANGES_REQUESTED** [review]

File: store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java [hunk 3]
```diff
@@ -59,8 +57,31 @@ public Void _doOperation(final Operation operation, final Context context, final
     @Override
     public FieldDeclaration getFieldDeclaration() {
         return new FieldDeclaration()
-                .fieldRequired(INPUT, Iterable.class)
+                .inputRequired(Iterable.class)
```
Comment (r32575) [inline store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java:60]: Changed to `inputRequired`

File: store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java [hunk 4]
```diff
@@ -59,8 +57,31 @@ public Void _doOperation(final Operation operation, final Context context, final
...
                 .fieldOptional(VALIDATE, Boolean.class)
                 .fieldOptional(SKIP_INVALID_ELEMENTS, Boolean.class);
     }
+
+    static class OperationBuilder extends BuilderSpecificInputOperation<OperationBuilder, AddElementsHandler> {
```
Comment (r32575) [inline store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/handler/AddElementsHandler.java:65]: Added the builder... Do the methods look okay?

**Review by GCHQDev404 — APPROVED** [review]

**Pull request merged**